Fixes when panel repeat is removed by clearing the templating variables

daniellee opened the pull request:
The original panel should clear all the templating variables and
look like it did before repeat was selected. Fixes #3831.

Also cleans up after a row repeat is removed.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=3968) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/><hr/>**Daniel Lee** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account ([for further information, click here] (https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user)).

daniellee committed:
fix(dashboard): fixes when panel repeat with templating removed

    The original panel should clear all the templating variables and
    look like it did before repeat was selected. Fixes #3831. Also
cleans up after a row repeat is removed

torkelo commented:
looks great, :heart:
